### PR TITLE
Updates to maipu driver

### PR DESCRIPTION
**Pull request by ktbyers**
Fixes rebase issue
Fixes linting issue.

### CODE REVIEW

File: netmiko/maipu/maipu.py [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+        # Clear the read buffer
+        time.sleep(0.3 * self.global_delay_factor)
+        self.clear_buffer()
+
+    def config_mode(
+        self,
+        config_command: str = "configure terminal",
+        pattern: str = "",
+        re_flags: int = 0,
+    ) -> str:
```
Comment (ktbyers) [inline netmiko/maipu/maipu.py:22]: This looks exactly the same as the parent class? I don't see any reason/need for it?

File: netmiko/maipu/maipu.py [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+        """Enter configuration mode."""
+        return super().config_mode(
+            config_command=config_command, pattern=pattern, re_flags=re_flags
+        )
+
+    def check_config_mode(
+        self, check_string: str = ")#", pattern: str = "#", force_regex: bool = False
+    ) -> bool:
```
Comment (ktbyers) [inline netmiko/maipu/maipu.py:30]: This looks exactly the same as the parent class? I don't see any reason/need for it?

File: netmiko/maipu/maipu.py [hunk 3]
```diff
@@ -0,0 +1,50 @@
...
+            cmd=cmd, confirm=confirm, confirm_response=confirm_response
+        )
+
+
+class MaipuSSH(MaipuBase):
+    pass
+
+
+class MaipuTelnet(MaipuBase):
+    pass
```
Comment (ktbyers) [inline netmiko/maipu/maipu.py:50]: Did you run the tests against a device for Telnet also?

File: tests/etc/responses.yml.example [hunk 4]
```diff
@@ -396,3 +396,12 @@ audiocode_shell:
   save_config: 'Configuration has been saved'
   cmd_response_init: "Test_String1"
   cmd_response_final: "Test_String2"
+
+maipu:
+  base_prompt: maipu_sw
+  router_prompt : maipu_sw>
+  enable_prompt: maipu_sw#
+  interface_ip: 11.11.11.11
+  version_banner: "MyPower (R) Operating System Software"
```
Comment (ktbyers) [inline tests/etc/responses.yml.example:405]: Do you know which versions of their OS this driver should work on?

**Comment by ktbyers** [comment]
@alizee2020 See comments above.

File: netmiko/maipu/maipu.py (hunk 1 above)
Comment (alizee2020) [reply netmiko/maipu/maipu.py:22]: yes,I will delete this method.

File: netmiko/maipu/maipu.py (hunk 2 above)
Comment (alizee2020) [reply netmiko/maipu/maipu.py:30]: yes,I will delete this method

File: netmiko/maipu/maipu.py (hunk 3 above)
Comment (alizee2020) [reply netmiko/maipu/maipu.py:50]: yes ,I did.
I will add the maipu_telnet in tests/etc

E:\netmiko\tests>py.test -v test_netmiko_show.py --test_device maipu_telnet
============================= test session starts =============================
platform win32 -- Python 3.8.0, pytest-7.1.3, pluggy-1.0.0 -- d:\python3.8\pytho
n.exe
cachedir: .pytest_cache
rootdir: E:\netmiko, configfile: setup.cfg
collected 25 items

test_netmiko_show.py::test_failed_key SKIPPED (Not using SSH-keys)       [  4%]
test_netmiko_show.py::test_disable_paging PASSED                         [  8%]
test_netmiko_show.py::test_terminal_width PASSED                         [ 12%]
test_netmiko_show.py::test_ssh_connect PASSED                            [ 16%]
test_netmiko_show.py::test_ssh_connect_cm PASSED                         [ 20%]
test_netmiko_show.py::test_send_command_timing PASSED                    [ 24%]
test_netmiko_show.py::test_send_command_timing_no_cmd_verify SKIPPED     [ 28%]
test_netmiko_show.py::test_send_command PASSED                           [ 32%]
test_netmiko_show.py::test_send_command_no_cmd_verify SKIPPED            [ 36%]
test_netmiko_show.py::test_complete_on_space_disabled SKIPPED            [ 40%]
test_netmiko_show.py::test_send_command_textfsm SKIPPED (TextFSM/ntc...) [ 44%]
test_netmiko_show.py::test_send_command_ttp SKIPPED (TTP template no...) [ 48%]
test_netmiko_show.py::test_send_command_genie SKIPPED (Genie not sup...) [ 52%]
test_netmiko_show.py::test_send_multiline_timing SKIPPED                 [ 56%]
test_netmiko_show.py::test_send_multiline SKIPPED                        [ 60%]
test_netmiko_show.py::test_send_multiline_prompt SKIPPED                 [ 64%]
test_netmiko_show.py::test_send_multiline_simple SKIPPED                 [ 68%]
test_netmiko_show.py::test_base_prompt PASSED                            [ 72%]
test_netmiko_show.py::test_strip_prompt PASSED                           [ 76%]
test_netmiko_show.py::test_strip_command PASSED                          [ 80%]
test_netmiko_show.py::test_normalize_linefeeds PASSED                    [ 84%]
test_netmiko_show.py::test_clear_buffer PASSED                           [ 88%]
test_netmiko_show.py::test_enable_mode PASSED                            [ 92%]
test_netmiko_show.py::test_disconnect PASSED                             [ 96%]
test_netmiko_show.py::test_disconnect_no_enable SKIPPED                  [100%]

=========================== short test summary info ===========================
SKIPPED [1] test_netmiko_show.py:25: Not using SSH-keys
SKIPPED [1] test_netmiko_show.py:88: Skipped
SKIPPED [1] test_netmiko_show.py:106: Skipped
SKIPPED [1] test_netmiko_show.py:128: Skipped
SKIPPED [1] test_netmiko_show.py:149: TextFSM/ntc-templates not supported on thi
s platform
SKIPPED [1] test_netmiko_show.py:171: TTP template not existing for this platfor
m
SKIPPED [1] test_netmiko_show.py:211: Genie not supported on this platform
SKIPPED [1] test_netmiko_show.py:231: Skipped
SKIPPED [1] test_netmiko_show.py:247: Skipped
SKIPPED [1] test_netmiko_show.py:272: Skipped
SKIPPED [1] test_netmiko_show.py:296: Skipped
SKIPPED [1] test_netmiko_show.py:401: Skipped
======================= 13 passed, 12 skipped in 22.36s =======================

E:\netmiko\tests>py.test -v test_netmiko_config.py --test_device maipu_telnet
============================= test session starts =============================
platform win32 -- Python 3.8.0, pytest-7.1.3, pluggy-1.0.0 -- d:\python3.8\pytho
n.exe
cachedir: .pytest_cache
rootdir: E:\netmiko, configfile: setup.cfg
collected 13 items

test_netmiko_config.py::test_ssh_connect PASSED                          [  7%]
test_netmiko_config.py::test_enable_mode PASSED                          [ 15%]
test_netmiko_config.py::test_config_mode PASSED                          [ 23%]
test_netmiko_config.py::test_exit_config_mode PASSED                     [ 30%]
test_netmiko_config.py::test_config_set PASSED                           [ 38%]
test_netmiko_config.py::test_config_set_generator PASSED                 [ 46%]
test_netmiko_config.py::test_config_set_longcommand PASSED               [ 53%]
test_netmiko_config.py::test_config_hostname PASSED                      [ 61%]
test_netmiko_config.py::test_config_from_file SKIPPED                    [ 69%]
test_netmiko_config.py::test_config_error_pattern SKIPPED (No error_...) [ 76%]
test_netmiko_config.py::test_banner SKIPPED (No banner defined.)         [ 84%]
test_netmiko_config.py::test_global_cmd_verify SKIPPED (No banner de...) [ 92%]
test_netmiko_config.py::test_disconnect PASSED                           [100%]

=========================== short test summary info ===========================
SKIPPED [1] test_netmiko_config.py:163: Skipped
SKIPPED [1] test_netmiko_config.py:175: No error_pattern defined.
SKIPPED [1] test_netmiko_config.py:209: No banner defined.
SKIPPED [1] test_netmiko_config.py:242: No banner defined.
======================== 9 passed, 4 skipped in 6.06s =========================

File: tests/etc/responses.yml.example (hunk 4 above)
Comment (alizee2020) [reply tests/etc/responses.yml.example:405]: This driver can work on  most of theit OS. I will change the desction as Version Information.

**Comment by alizee2020** [comment]
@ktbyers Hi,ktbyers.I have replied the above comments. And make some change in my code . I had pulled a new request #2992 . Please help me review again. Thank so much.

**Comment by ktbyers** [comment]
Replaced by https://github.com/ktbyers/netmiko/pull/2992

**Pull request closed without merging**